Mathlib/Analysis/Convex/Combination.lean

```diff
@@ -551,3 +558,29 @@ theorem AffineBasis.convexHull_eq_nonneg_coord {ι : Type*} (b : AffineBasis ι
     rw [b.coord_apply_combination_of_mem hi hw₁] at hx
     exact hx
 #align affine_basis.convex_hull_eq_nonneg_coord AffineBasis.convexHull_eq_nonneg_coord
+
+variable {s t t₁ t₂ : Finset E}
+
+lemma AffineIndependent.convexHull_inter (hs : AffineIndependent R ((↑) : s → E))
```

Comment:
`hs` seems like a rather strong assumption; the lemma is still true for `t₁ = {(0, 0), (0, 1), (0, 2)}` and `t₂ = {(0, 0), (1, 0), (2, 0)}`, right?

Reply:
It is in general very wrong. Consider the intersection of two random triangles in the plane: it might be an hexagon!

This condition is the gluing condition in the definition of `Geometry.SimplicialComplex`. You can't expect it to be true in much greater generality than what I have here.

Reply:
I think `hs` is sufficiently weird here that a docstring is justified, and maybe a longer lemma name (`convexHull_inter_of_subset`?)

Reply:
It's in the `AffineIndependent` namespace! I thought this would be enough to disambiguate.